includes/definitions/discovery/baicells-od04.yaml

```diff
@@ -0,0 +1,127 @@
+mib: BAICELLS-CPE-MIB
+modules:
+    os:
+        hardware: BAICELLS-CPE-MIB::HardwareVersion.0
+        serial: BAICELLS-CPE-MIB::SerialNumber.0
+        version: BAICELLS-CPE-MIB::SoftwareVersion.0
+    sensors:
+        runtime:
+            data:
+                -
+                    oid: UpTime
+                    num_oid: .1.3.6.1.4.1.53058.2.1.9.0
+                    descr: 'Uptime'
+                    index: 'UpTime.0'
+                    divisor: 60
+                # TODO write a function to convert xx:yy:zz to minutes
+                #-
+                #    oid: ConnectionTime
+                #    num_oid: .1.3.6.1.4.1.53058.2.3.5.0
+                #    index: 'ConnectionTime'
+                #    descr: 'Connection time'
+        dbm:
+            data:
+                -
+                    oid: AntRSRP0
+                    num_oid: .1.3.6.1.4.1.53058.2.2.7.0
```

Comment:
All these num_oid fields are missing {{ $index }} as indicated by the test failures.

Reply:
Correct, but as I mentioned in my PR `{{ $index }}` is not substituted as one would expect; it becomes an empty string, so in this example it becomes ` num_oid: '.1.3.6.1.4.1.53058.2.2.7.'`.

I am not totally sure why this happening, but I think it could be because the MIB is not using tables (SEQUENCE types ?) anywhere. Instead, all the OIDs in the MIB are OBJECT TYPE at the same hierarchical level. Other MIBs I've checked look pretty different.

Let me know if you have more insight on this, I've spent many hours trying to make it work with `{{ $index }}` unsuccessfully.